Sources/UI/conversationlist/DJLConversationListViewController.mm

```diff
@@ -917,7 +917,8 @@ - (void) djl_tableViewResignFirstResponder:(NSTableView *)tableView
 
 - (NSMenu *) djl_tableView:(NSTableView *)tableView menuForEvent:(NSEvent *)event row:(NSInteger)row
 {
-    if ([[_tableView selectedRowIndexes] count] == 0) {
+    // Select right-clicked row if it isn't already in the selection.
+    if ([[_tableView selectedRowIndexes] containsIndex:row] == NO) {
```

Comment:
```
if (![[_tableView selectedRowIndexes] containsIndex:row]) {
```

Reply:
Sure. Just so I don't mess this up again: I'm going to make a new commit with this change on my local branch and then push it to Github and it will update this pull request?

Reply:
yup!